### PR TITLE
feat(login): add branding to login redirect page

### PR DESCRIPTION
**Pull request by craicoverflow**
This PR includes the new product branding into the login web page.

### Verification Steps

1. Run the `login` page
2. The login web page should include the new branding on the main body and as a favicon.

image

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### Checklist

- [x] Documentation added for the feature
- [x] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer

### CODE REVIEW

File: pkg/auth/login/static/mas-sso-redirect-page.html [hunk 1]
```diff
@@ -3,12 +3,13 @@
...
   <link rel="stylesheet" href="https://unpkg.com/@patternfly/patternfly@4.70.2/patternfly.css">
   <title>%v</title>
+  <link rel="icon" type="image/x-icon" href="/static/img/favicon.ico"/>
 </head>
 
 <body>
   <div class="pf-c-empty-state">
     <div class="pf-c-empty-state__content">
-      <i class="fas fa-key pf-c-empty-state__icon" aria-hidden="true"></i>
+      <img src="/static/img/logo.png" alt="Logo"/>
```
Comment (craicoverflow) [inline pkg/auth/login/static/mas-sso-redirect-page.html:12]: I will change these to SVG

Comment (wtrocki) [reply pkg/auth/login/static/mas-sso-redirect-page.html:12]: Yes. Inline will be easier than Embed

**Review by wtrocki — APPROVED** [review]
Verified

File: pkg/auth/login/static/mas-sso-redirect-page.html (hunk 1 above)
Comment (craicoverflow) [reply pkg/auth/login/static/mas-sso-redirect-page.html]: What? I will still need to embed them.

**Pull request merged**